formats/src/main/scala/overflowdb/formats/graphson/GraphSONImporter.scala

```diff
@@ -0,0 +1,45 @@
+package overflowdb.formats.graphson
+
+import overflowdb.Graph
+import overflowdb.formats.Importer
+import overflowdb.formats.graphson.GraphSONProtocol._
+import spray.json._
+
+import java.nio.file.Path
+import scala.io.Source.fromFile
+
+/**
+ * Imports OverflowDB graph to GraphSON 3.0
+ *
+ * https://tinkerpop.apache.org/docs/3.4.1/dev/io/#graphson-3d0
+ */
+object GraphSONImporter extends Importer {
+
+  override def runImport(graph: Graph, inputFiles: Seq[Path]): Unit = {
+    assert(inputFiles.size == 1, s"input must be exactly one file, but got ${inputFiles.size}")
+    val source = fromFile(inputFiles.head.toFile)
+    val graphSON = try source.mkString.parseJson.convertTo[GraphSON]
```

Comment:
for the small scope here this is fine, but in general `Using` is the better alternative for ARM (automatic resource management): `Using.resource`

https://www.scala-lang.org/api/current/scala/util/Using$.html